source/plugin/scheduler.rst

```diff
@@ -93,6 +114,34 @@ To cancel a task, simply call the ``Task#cancel`` method:
 
 	task.cancel();
 
+If you need to cancel the task from within the runnable itself, you can instead opt to use a `Consumer<Task>` in order to
+access the task. The below example will schedule a task that will count down from 60 and cancel itself upon reaching 0.
+
+.. code-block:: java
+
+        Scheduler scheduler = Sponge.getScheduler();
+        Task.Builder taskBuilder = scheduler.createTaskBuilder();
+
+        Task task = taskBuilder.execute(new CancellingTimerTask())
+            .interval(1, TimeUnit.SECONDS)
+            .name("Self-Cancelling Timer Task").submit(plugin);
+    }
```

Comment:
That brace is the ending of the method that was initializing the class, before the inner class was defined. I wasn't sure to include the brace or not, or the method definition or not.